README.md

```diff
@@ -46,19 +46,21 @@ Development environment supports hot reload with Webpack's [Hot Module Replaceme
 
 ### With Cryostat
 
-First, launch a [`Cryostat`](https://github.com/cryostatio/cryostat) instance with CORS enabled by setting `CRYOSTAT_CORS_ORIGIN` to `http://localhost:9000`. For example:
-
+First, launch a [`Cryostat`](https://github.com/cryostatio/cryostat) instance:
 ```bash
 $ cd /path/to/cryostat
-$ CRYOSTAT_DISABLE_SSL=true CRYOSTAT_CORS_ORIGIN=http://localhost:9000 sh run.sh
+$ bash smoketest.bash
 ```
 
 Then, run:
 
 ```bash
-$ yarn start:dev
+# By default, CRYOSTAT_URL is set to http://localhost:8080
+$ CRYOSTAT_URL=https://localhost:8443 yarn start:dev 
```

Comment:
After https://github.com/cryostatio/cryostat/pull/426 goes in, HTTPS will be default on the Cryostat smoketest, so I think it makes sense that this PR should also default to HTTPS and let it be disabled by setting `CRYOSTAT_URL`.

Also, should this be `CRYOSTAT_PROXY_URL` like in line 24 of `webpack.dev.js`?

Reply:
Ah yess sounds good!

> Also, should this be CRYOSTAT_PROXY_URL like in line 24 of webpack.dev.js?

Opps, I copied the wrong command. Thanks!